public/index.html

```diff
@@ -1,34 +1,37 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
```

Comment:
Why is the index.html being edited in this PR?

Reply:
So I wanted to add bootstrap and that required adding some script tags but it didn't work so I deleted the tags that I added, I don't believe any changes other than whitespace were made. In the future how would I go about preventing changes in whitespace showing up in my changes?